### PR TITLE
166538796 reorder week days in service view

### PR DESCRIPTION
**Pull request by rckopteri**
# Added
* Add function that reorders week day set. And take it into use in service-viewer page

### CODE REVIEW

File: ote/src/cljc/ote/util/transport_service_util.cljc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+(def week-day-order {:MON 1, :TUE 2, :WED 3, :THU 4, :FRI 5, :SAT 6, :SUN 7})
+
+;; Reorder week days mon 1, tue 2, wed 3, thu 4, fri 5, sat 6, sun 7
+(defn reorder-week-days
+  "Receives set of week days. Returns list of week days to make sure that ordering remains correct."
+  [week-day-set]
+  (let [ordered-days (sort (mapv #(% week-day-order) week-day-set))
+        ordered-days->list (map
+                             (fn [od]
+                               (first
```
Comment (jkorpelin) [inline ote/src/cljc/ote/util/transport_service_util.cljc]: double `(first)` call could be replaced with `(second)`?

**Comment by jkorpelin** [comment]
I'm not sure if this is needed since the days could be sorted when saved? And the days are sorted in the form where they are added, could the same method be used in the all info page?

**Comment by rckopteri** [comment]
Response for Joni: If we are dependent on the order where week days are saved, then the ordering should be in the back end when they are saved. Also we should reorder all days that are already in the database.
So the simplest way is to reorder them when they are shown.

File: ote/src/cljs/ote/views/service_viewer.cljs [hunk 2]
```diff
@@ -466,7 +467,8 @@
              (tr [:service-viewer :day-of-week])
              (string/join ", " (map
                                  #(string/lower-case (tr [:enums ::t-service/day :short %]))
-                                 (::t-service/week-days time))) true]]
+                                 (tsu/reorder-service-days (::t-service/week-days time))))
```
Comment (philosotractor) [inline ote/src/cljs/ote/views/service_viewer.cljs]:
```suggestion
                                 (tsu/reorder-week-days (::t-service/week-days time))))
```
?

Comment (rckopteri) [reply ote/src/cljs/ote/views/service_viewer.cljs]: hups. Thanx!

**Pull request merged**